src/backend/routers/students.test.ts

```diff
@@ -25,6 +26,7 @@ test("getAllStudents", async (t) => {
     .values({
       first_name: "Foo",
       last_name: "Bar",
+      email: "foo.bar@email.com",
```

Comment:
It would be good to add a test case that tries to insert a duplicate email into the db.

Reply:
I added the new test case for duplicate emails. Please take a look and also let me know if the naming is accurate to describe the test.